doc/benchmarking.md

```diff
@@ -2,54 +2,131 @@ Benchmarking
+DuplicateInputs, 5, 10, 0.349838, 0.00681664, 0.0071578, 0.00699272
+FastRandom_1bit, 5, 440000000, 3.55037, 1.60107e-09, 1.63897e-09, 1.60388e-09
+FastRandom_32bit, 5, 110000000, 4.66069, 8.43174e-09, 8.50173e-09, 8.47942e-09
+MatchGCSFilter, 5, 50000, 5.29045, 2.10328e-05, 2.12565e-05, 2.11911e-05
+MempoolEviction, 5, 41000, 3.02568, 1.46642e-05, 1.49587e-05, 1.47037e-05
+MerkleRoot, 5, 800, 4.55175, 0.00113359, 0.00114235, 0.00113847
+POLY1305_1MB, 5, 340, 1.20259, 0.000703508, 0.000711726, 0.000707535
+POLY1305_256BYTES, 5, 250000, 0.240919, 1.87676e-07, 2.0346e-07, 1.9024e-07
+POLY1305_64BYTES, 5, 500000, 0.14503, 5.73722e-08, 5.9313e-08, 5.78021e-08
+PrePadded, 5, 10000, 0.0105583, 2.08606e-07, 2.16661e-07, 2.10247e-07
+PrevectorClearNontrivial, 5, 28300, 9.74354, 6.7409e-05, 7.11888e-05, 6.83039e-05
+PrevectorClearTrivial, 5, 88600, 19.8132, 4.43881e-05, 4.52573e-05, 4.46334e-05
+PrevectorDeserializeNontrivial, 5, 6800, 2.48919, 7.19162e-05, 7.49433e-05, 7.2365e-05
+PrevectorDeserializeTrivial, 5, 52000, 2.75915, 1.04626e-05, 1.07667e-05, 1.06177e-05
+PrevectorDestructorNontrivial, 5, 28800, 9.03402, 6.1415e-05, 6.52106e-05, 6.16662e-05
+PrevectorDestructorTrivial, 5, 88900, 18.6047, 4.16076e-05, 4.21505e-05, 4.18157e-05
+PrevectorResizeNontrivial, 5, 28900, 2.88001, 1.98667e-05, 2.00179e-05, 1.98966e-05
+PrevectorResizeTrivial, 5, 90300, 3.24412, 7.16336e-06, 7.25998e-06, 7.16659e-06
+RIPEMD160, 5, 440, 4.88606, 0.00220545, 0.00225396, 0.00221235
+RegularPadded, 5, 10000, 0.0211132, 4.07576e-07, 4.53654e-07, 4.11425e-07
+RollingBloom, 5, 1500000, 3.65431, 4.76011e-07, 5.10435e-07, 4.80835e-07
+RollingBloomReset, 5, 20000, 5.59406, 5.52469e-05, 5.76053e-05, 5.57199e-05
+RpcMempool, 5, 40, 3.17675, 0.0158117, 0.0160428, 0.0158462
+SHA1, 5, 570, 4.70379, 0.0016439, 0.0016631, 0.00164822
+SHA256, 5, 340, 4.74761, 0.00277871, 0.00280425, 0.00279286
+SHA256D64_1024, 5, 7400, 4.57789, 0.000118055, 0.000130664, 0.000122628
+SHA256_32b, 5, 4700000, 4.74901, 1.98897e-07, 2.05686e-07, 2.01991e-07
+SHA512, 5, 330, 4.56722, 0.00273604, 0.00279254, 0.00276721
+SipHash_32b, 5, 40000000, 5.00388, 2.46464e-08, 2.55899e-08, 2.48747e-08
+Sleep100ms, 5, 10, 5.01005, 0.100187, 0.10021, 0.100201
+Trig, 5, 12000000, 1.72352, 2.60512e-08, 3.09944e-08, 2.8639e-08
+VerifyScriptBench, 5, 6300, 4.28922, 0.000132975, 0.000143875, 0.000133716
+WalletBalanceClean, 5, 8000, 2.32476, 5.75268e-05, 5.96201e-05, 5.77432e-05
+WalletBalanceDirty, 5, 2500, 3.20313, 0.000255601, 0.000256807, 0.000256311
+WalletBalanceMine, 5, 16000, 2.2409, 2.78764e-05, 2.81732e-05, 2.8009e-05
+WalletBalanceWatch, 5, 8000, 2.29993, 5.73364e-05, 5.77806e-05, 5.74599e-05
 ```
 
 Help
 ---------------------
-`-?` will print a list of options and exit:
+```
+$ src/bench/bench_bitcoin -?
+
+Options:
+
+  -?
+       Print this help message and exit
+
+  -evals=<n>
+       Number of measurement evaluations to perform. (default: 5)
```

Comment:
Same here. This will just end up being outdated in two weeks

Reply:
Not adding the `bench_bitcoin` options ? Was thinking it's good to see options without compiling binary

Reply:
They just get outdated and incorrect information is worse than none, no?